thirdeye-server/src/main/java/ai/startree/thirdeye/auth/AuthorizationManager.java

```diff
@@ -145,18 +155,16 @@ public boolean hasRootAccess(final ThirdEyePrincipal principal) {
   // Returns the resource identifier for a dto.
   // Null is ok and maps to a default resource id.
   public ResourceIdentifier resourceId(final AbstractDTO dto) {
-    final var name = optional(dto)
+    final var name = Optional.ofNullable(dto)
```

Comment:
This is the same thing. didn't understand this

Reply:
Oh ya it is, just feels a little unnecessary to use the wrapper. I can change it back.